### PR TITLE
Documentation Improvement

### PR DESCRIPTION
**Pull request by sturkmen72**
documentation improvement

ready to review
```
force_builders_only=docs
```

### CODE REVIEW

**Comment by sturkmen72** [comment]
@Tetragramm could you check sentence changes in this PR

File: modules/highgui/include/opencv2/highgui.hpp [hunk 1]
```diff
@@ -507,7 +516,7 @@ CV_EXPORTS_W Rect selectROI(const String& windowName, InputArray img, bool showC
 CV_EXPORTS_W Rect selectROI(InputArray img, bool showCrosshair = true, bool fromCenter = false);
 
 /** @brief Selects ROIs on the given image.
-Function creates a window and allows user to select a ROIs using mouse.
+The function creates a window and allows the user to select a ROIs using mouse.
```
Comment (Tetragramm) [inline modules/highgui/include/opencv2/highgui.hpp:519]: Would change to "...select multiple ROIs..."

**Comment by Tetragramm** [comment]
Everything else looks good.  
If we want to be extremely pedantic, we can do "using the mouse." on the two selectROI functions, but it's perfectly understandable as is.

File: modules/highgui/include/opencv2/highgui.hpp (hunk 1 above)
Comment (sturkmen72) [reply modules/highgui/include/opencv2/highgui.hpp:519]: maybe "select a ROI" AFAIK selectROI allows to select a single ROI

**Comment by sturkmen72** [comment]
@Tetragramm Thank you very much.

**Comment by sturkmen72** [comment]
@asmorkalov btw what does "WIP" mean?

**Comment by asmorkalov** [comment]
@sturkmen72 Is the patch ready for review?

**Comment by sturkmen72** [comment]
@asmorkalov yes, the patch is ready for review.

**Comment by asmorkalov** [comment]
Looks good to me :+1: 
IMHO, `the mouse` and `the user` are redundant. Just `mouse` and `user` sounds better.

**Review by asmorkalov — APPROVED** [review]

**Comment by sturkmen72** [comment]
@asmorkalov before correcting white space problem let me know if adding link  is good or not.
[preview](https://pullrequest.opencv.org/buildbot/export/pr/18675/docs/d7/dfc/group__highgui.html#ga8daf4730d3adf7035b6de9be4c469af5)

**Comment by asmorkalov** [comment]
https://www.learnopencv.com/how-to-select-a-bounding-box-roi-in-opencv-cpp-python/ is external resource. I prefer not not add external links besides white papers, external projects, standards, etc.

**Comment by sturkmen72** [comment]
@asmorkalov thank you for reviewing.

>The function creates a window and allows users to select a ROI using the mouse.
>The function creates a window and allows users to select multiple ROIs using the mouse.

i hope this sounds better.

**Pull request merged**